phoenix-ios/phoenix-ios/sync/SyncSeedManager.swift

```diff
@@ -350,9 +576,9 @@ class SyncSeedManager: SyncManagerProtcol {
 						recordID: recordID()
 					)
 					
-					record[record_column_mnemonics] = recoveryPhrase.mnemonics
-					record[record_column_language] = recoveryPhrase.languageCode
 					record[record_column_name] = uploadedName
+					record[record_column_language] = recoveryPhrase.languageCode
+					record.encryptedValues[record_column_mnemonics_encrypted] = recoveryPhrase.mnemonics
```

Comment:
What happens if the user has not enabled "Advanced data protection"? Does `CKRecord.encryptedValues` falls back to a non-encrypted value?